code-challenges.js

```diff
@@ -40,24 +79,72 @@ var vowelTester1 = "learn"
 // Expected output: 1
 var vowelTester2 = "throw"
 // Expected output: 3
-
-
-
-
-
+//Create a function that iterates each letters
+//have another function intandem to return the index of only the first vowel indexOf()
+//turn string into array of letters
+//for loop that checks AEIOU
+//return index
+var arrayVowel = ["e" , "a" , "o" , "u" , "i"]
```

Comment:
This variable should be within the function scope, so that way it can be used anywhere, not just floating in the code ether ;)